tests/cypress/support/step_definitions/formInputs.js

```diff
@@ -1,11 +1,83 @@
 import { When } from "@badeball/cypress-cucumber-preprocessor";
 When("these form elements are filled:", (dataTable) => {
   dataTable.rawTable.forEach((row) => {
-    cy.get(`[name='${row[0]}']`).type(row[1]);
+    switch (row[1]) {
+      case "singleCheckbox":
+        if (row[2] == "true") cy.get(`[name='${row[0]}']`).check();
+        else cy.get(`[name='${row[0]}']`).uncheck();
+        break;
+      case "radio":
+        cy.get(`[name='${row[0]}']`).check(row[2]);
+        break;
+      case "checkbox":
+        cy.get(`[name='${row[0]}']`).check(row[2]);
+        break;
+      case "dropdown":
+        cy.get(`[name='${row[0]}']`).select(row[2]);
+        break;
+      case "repeated":
+        cy.get(`input[name^="${row[0]}"]`).eq(row[3]).type(row[2]);
+        break;
+      default:
+        cy.get(`[name='${row[0]}']`).type(row[2]);
+        break;
+    }
+  });
+});
+
+When("these form elements are edited:", (dataTable) => {
```

Comment:
Is this step def significantly different that form elements filled?  I think it is some elements are cleared, but most are the same. Might be an opportunity for future improvement. 
I think we can simplify this by calling `.clear()` on the appropriate items, then using `Step` to call `these form elements are filled:`

see [these docs for more info](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/cucumber-basics.md#nested-steps)

Reply:
this is a good suggestion! I'll work on it.

Reply:
Updated and talked with Britt and Thomas about solution :)